XenModel/FriendlyNames.resx

```diff
@@ -1835,4 +1835,7 @@
   <data name="Message.action-pbd_plug_failed_on_server_start" xml:space="preserve">
     <value>Repair Storage</value>
   </data>
+  <data name="Label-host.edition-desktop-cloud" xml:space="preserve">
+    <value>XenApp/XenDesktop [Citrix] Cloud</value>

```

Comment:
Just wondering if we should use "XenApp/XenDesktop [Citrix] Cloud" or "XenApp/XenDesktop [Citrix Cloud]". If the branding info is replaced for some reason, should "Cloud" also be replaced?

Reply:
We discussed this and decided to keep it as [Citrix] Cloud